client/gutenberg/extensions/business-hours/edit.jsx

```diff
@@ -68,26 +77,24 @@ class BusinessHours extends Component {
 				<dl className={ classNames( className, 'jetpack-business-hours' ) }>
 					{ localizedWeek.map( ( day, key ) => {
 						return (
-							<DaySave key={ key } day={ day } localization={ localization } timeFormat={ time } />
+							<DayPreview
+								key={ key }
+								day={ day }
+								localization={ localization }
+								timeFormat={ time }
+							/>
 						);
 					} ) }
 				</dl>
 			);
 		}
 
-		if ( ! hasFetched ) {
-			return (
-				<Placeholder
-					icon={ <BlockIcon icon={ icon } /> }
-					label={ __( 'Loading business hours' ) }
-				/>
-			);
-		}
-
 		return (
 			<div className={ classNames( className, 'is-edit' ) }>
 				{ localizedWeek.map( ( day, key ) => {
-					return <Day key={ key } day={ day } localization={ localization } { ...this.props } />;
+					return (
+						<DayEdit key={ key } day={ day } localization={ localization } { ...this.props } />
```

Comment:
A minor point: I prefer to avoid `...spreadProps` in most Component hierarchies. I find the clarity of explicit prop passing to be more maintainable.